supabase/migrations/20260814212824_user_onboarding_progress.sql

```
@@ -0,0 +1,18 @@
+ALTER TABLE "public"."users"
+ADD COLUMN "onboarding" "jsonb" DEFAULT '{}'::"jsonb" NOT NULL;
+
+COMMENT ON COLUMN "public"."users"."onboarding" IS 'Persisted create-app onboarding wizard progress for resume and admin drop-off. Keys: status, step, flow, intent, app_name, app_id, existing_app, existing_app_setup, store_url, imported_store_app_id, org_name, estimated_users_index, updated_at, completed_at.';
+
+ALTER TABLE "public"."users"
+ADD CONSTRAINT "users_onboarding_valid" CHECK (
+  ("jsonb_typeof"("onboarding") = 'object'::"text")
+  AND ("octet_length"(("onboarding")::"text") <= 8192)
+  AND ((NOT ("onboarding" ? 'status'::"text")) OR (("onboarding" ->> 'status'::"text") = ANY (ARRAY['in_progress'::"text", 'completed'::"text", 'abandoned'::"text"])))
```

SQLFluff (4.2.2)

[error] Line is too long (167 > 80).

(LT05)